cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -516,7 +523,7 @@ func generateInitramfsMounts() error {
 }
 
 func unlockIfEncrypted(name string) (string, error) {
-	device := filepath.Join("/dev/disk/by-label", name)
+	device := filepath.Join(diskByLabel, name)
```

Comment:
fwiw this could also be under dirs.GlobalRootDir too and we could accomplish the same thing in the tests without another Mock* in export_test.go